### PR TITLE
feat(ui5-avatar): introduce initials, imageFitType and backgroundColo…

### PR DESCRIPTION
**Pull request by ivoplashkov**
initials, imageFitType and backgroundColor properties have been introduced.

### CODE REVIEW

File: packages/main/src/Avatar.js [hunk 1]
```diff
@@ -82,6 +96,29 @@ const metadata = {
 			type: String,
 			defaultValue: AvatarSize.S,
 		},
+
+		/**
+		 * Defines the fit type of the desired image.
```
Comment (ilhan007) [inline packages/main/src/Avatar.js]: add info for the available options are: see the "size" property docs for reference.

File: packages/main/src/Avatar.js [hunk 2]
```diff
@@ -82,6 +96,29 @@ const metadata = {
...
+		 */
+		imageFitType: {
+			type: String,
+			defaultValue: AvatarFitType.Cover,
+		},
+
+		/**
+		 * Defines the background color of the desired image.
+		 * @type {String}
+		 * @defaultvalue "Accent6"
```
Comment (ilhan007) [inline packages/main/src/Avatar.js]: add info for the available options are: see the "size" property docs for reference.

File: packages/main/src/themes/Avatar.css [hunk 3]
```diff
@@ -85,8 +91,56 @@
...
+:host([backgroundColor="Accent7"]) {
+	background-color: var(--sapAccentColor7);
+}
+
+:host([backgroundColor="Accent8"]) {
+	background-color: var(--sapAccentColor8);
+}
+
+:host([backgroundColor="Accent9"]) {
+	background-color: red;
```
Comment (ilhan007) [inline packages/main/src/themes/Avatar.css]: this is something you missed I guess

File: packages/main/src/Avatar.js (hunk 1 above)
Comment (ivoplashkov) [reply packages/main/src/Avatar.js]: Done.

File: packages/main/src/Avatar.js (hunk 2 above)
Comment (ivoplashkov) [reply packages/main/src/Avatar.js]: Done.

File: packages/main/src/themes/Avatar.css (hunk 3 above)
Comment (ivoplashkov) [reply packages/main/src/themes/Avatar.css]: Done.

**Pull request merged**